File: app/src/main/java/de/nicidienase/chaosflix/ChaosflixApplication.java
package de.nicidienase.chaosflix;

import android.app.Application;
import android.content.Context;
import android.support.multidex.MultiDex;

import com.facebook.stetho.Stetho;
import com.orm.SugarContext;

/**
 * Created by felix on 18.03.17.
 */

public class ChaosflixApplication extends Application {
	@Override
	public void onCreate() {
		super.onCreate();
		Stetho.initializeWithDefaults(this);
		SugarContext.init(this);
	}

	@Override
	public void onTerminate() {
		SugarContext.terminate();
		super.onTerminate();
	}

	@Override
	protected void attachBaseContext(Context base) {
		MultiDex.install(base);
		super.attachBaseContext(base);
	}
}

File: touch/src/debug/java/de/nicidienase/chaosflix/touch/ChaosflixApplication.kt
package de.nicidienase.chaosflix.touch

import android.app.Application
import android.content.Context
import com.facebook.stetho.Stetho

class ChaosflixApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Stetho.initializeWithDefaults(this)
    }
}
